docs/installation-and-operations/system-requirements/README.md

```diff
@@ -37,11 +37,11 @@ There may come a point where you will have to make configuration changes to the
 
 Using a rough estimate we can give the following recommendations based on the number of total users.
 
-| Users | CPU cores | RAM in GB  | web workers | background workers | disk space in GB |
-|-------|-----------|------------|-------------|--------------------|------------------|
-| <=200 | 4         | 4          | 4           | 1                  | 20               |
-| 500   | 8         | 8          | 8           | 2                  | 40               |
-| 1500  | 16        | 16         | 16          | 4                  | 80               |
+| Total active users | CPU cores | RAM in GB | web workers | background workers | disk space in GB |
```

Comment:
I assume you changed this to mean something closer to concurrent users rather than simply the number of users in the database. Perhaps we should qualify this as such as well.